### PR TITLE
support of custom headers and return result to post table 

### PR DESCRIPTION
**Pull request by grigoryvasiliev**
Feature status: Alpha
Existing tests are passed

### CODE REVIEW

File: http_calls.py [hunk 1]
```diff
@@ -431,17 +501,36 @@ def isUndefined(self, val):
...
             for h in header:
                 if h == "_id?":
                     setattr(self, "_id", lambda self=self: self.ids.pop(0))
                 else:
-                    setattr(self, "set%s" % str.replace(h, h[0], h[0].upper(), 1), lambda x: x)
+                    setattr(
+                        self,
+                        "set%s" %
+                        str.replace(
+                            h,
```
Comment (AleksandrIakhnev) [inline http_calls.py:529]: Formatting looks not pretty, why do we need such one? Is it auto-formatted? This replace code is duplicated also.

Comment (mikeplavsky) [reply http_calls.py:529]: Duplication can not be removed until there are no tests

Comment (mikeplavsky) [reply http_calls.py:529]: imho formatting looks very nice! :)

**Comment by mikeplavsky** [comment]
Talked to @grigoryvasiliev and @AntonTimiskov - agreed to minimise number of empty cells - get rid of cell for re-running the query (performance tool).

**Comment by mikeplavsky** [comment]
btw now it is possible to provide headers through GET https://github.com/VolodymyrLavrenchuk/PySlim/blob/master/http_calls.py#L69 

imho it would be enough just to allow specifying default headers (like host https://github.com/VolodymyrLavrenchuk/PySlim/blob/master/http_calls.py#L76) This way all the previous functionality would be untouched.

**Pull request closed without merging**